frontends/web/src/locales/en/app.json

```diff
@@ -1027,6 +1027,7 @@
     },
     "successDisabled": {
       "message": "Optional passphrase <strong>successfully enabled</strong>!\nYou’ll be asked to provide a passphrase from now on.",
+      "messageEnd": "Please replug your BitBox02 now.",
```

Comment:
it could be someone else's bitbox02 :)
how about this: Please replug the BitBox02 device now.

Reply:
I agree, directly addressing the user can have a strong effect, i.e. when we really want to make them aware about something the should or shouldn't do.

But the effect is kind of gone if we always use direct addressing.

I will go through the text and propose some changes.